services/py-api/src/service/hackathon_service.py

```diff
@@ -65,6 +65,24 @@ async def create_random_participant(
         # As when first created, the random participant is not assigned to a team we return the team as None
         return Ok((result.ok_value, None))
 
+    async def verify_admin_participant_and_team_in_transaction(self, admin_id: str) -> Any:
```

Comment:
Please return Concrete types, not Any. Here, the type could be `Result[Tuple[Participant, Team], ParticipantNotFoundError | TeamNotFoundError | Exception` 

Also didn't you get an error from MyPy? It is set to [strict](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-strict) in its [config file](https://github.com/AUBGTheHUB/monolith/blob/main/services/py-api/.pre-commit-config.yaml): 
<img width="535" alt="image" src="https://github.com/user-attachments/assets/e71f3ccc-ea63-4f86-8615-f7e1792dddef" />

strict enables all optional flags including [--no-strict-optional](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-strict-optional)